repoclosure/yum_el7.conf

```diff
@@ -42,7 +42,7 @@ baseurl=http://mirror.centos.org/centos/7/sclo/x86_64/sclo/
 
 [el7-pl-prod]
```

Comment:
Do we use this name in the pipeline anywhere?

Reply:
We don't due to refactoring that was done. @dLobatog requested we just one-off run it but this change needs to be made

Reply:
I think we should rather be adding `[el7-puppet-4]`. Since we dropped puppet 3 we could safely drop `[el7-pl-{prod,deps}]` here.